src/workspace.cr

```diff
@@ -160,6 +168,14 @@ module Mint
           reset_cache
         end
       end
+
+      spawn do
+        @env_watcher.try(&.watch do
+          Env.load do
+            update_cache
+          end
+        end)
```

Comment:
```suggestion
        @env_watcher.try &.watch do
          Env.load do
            update_cache
          end
        end
```

Reply:
Oh, I thought the block would belong to the `try`, nice to know :D